modules/salt/pages/salt-gitfs.adoc

```diff
@@ -48,6 +48,17 @@ gitfs_remotes:
     - base: master
 ----
 
+top_file_merging_strategy:
+Define the strategy to merge multiple [literal]``top.sls`` files defined in your different fileserver roots.
+As a ``/srv/salt/top.sls`` file exists [literal]``merge_all`` is required to evaluate also the
+[literal]``top.sls`` file from the git repository.
++
+Example:
++
+----
+top_file_merging_strategy: merge_all
+----
```

Comment:
```suggestion
A [path]``/srv/salt/top.sls`` file already exists, so use [command]``merge_all`` to evaluate the [path]``top.sls`` file from the git repository.
For example:

----
top_file_merging_strategy: merge_all
----
```

Reply:
Not sure about the 'so' here. Maybe @keichwa has a better suggestion.

Reply:
Perhaps `therefore` ?

Reply:
it's wordy, but probably clearer.